lambdas/src/services/person-identity-service.ts

```diff
@@ -40,6 +41,7 @@ export class PersonIdentityService {
             birthDates: this.mapBirthDates(sharedClaims.birthDate),
             expiryDate: sessionExpirationEpoch,
             names: this.mapNames(sharedClaims.name),
+            socialSecurityRecord: this.mapNino(sharedClaims.socialSecurityRecord),
```

Comment:
Just thinking that this is always going to be an empty array in the NINo person details table, do you think we could optionally add it instead?

Reply:
Oh actually think I might be getting a bit confused with the different scenarios covered in the tests so we have

1. in person-identity-service.test `should avoid formatting blank identities` `socialSecurityRecord` is being passed in as an empty array and results in an empty array
2. in session-handler.test `should save to the session with no socialSecurityRecord included` `socialSecurityRecord` is absent 

I'm guessing that if the value is empty core will not send `socialSecurityRecord` at all in the shared claims, rather than an empty array is that correct?

Reply:
Yeah Core should only be sending fields required for each CRI